doc/maintaining/cli.rst

```diff
@@ -0,0 +1,572 @@
+.. _cli:
+
+======================
+Command Line Interface (CLI)
+======================
+
+NB: From CKAN 2.9 onwards the CKAN configuration file is named 'ckan.ini'. Previous names: 'production.ini', 'development.ini' plus others may also still appear in documentation and the software. These legacy names will eventually be phased out
+
+Most common CKAN administration tasks can be carried out from the command line
+on the server that CKAN is installed on, using the ``ckan`` command.
+
+If you have trouble running ckan commands, see
+`Troubleshooting ckan Commands`_ below.
+
+.. note::
+
+   Before running a CKAN ``ckan`` command, you have to activate your CKAN
+   virtualenv and change to the ``ckan``  directory, for example:
```

Comment:
Running commands from CKAN's folder is not required now. As long, as the virtual environment is active, `ckan` command is available everywhere in the host system, even at `/`.